src/inhibitors/cooldown.js

```diff
@@ -11,7 +11,15 @@ module.exports = class extends Inhibitor {
 
 		const existing = command.cooldowns.get(message.levelID);
 
-		if (existing && existing.limited) throw message.language.get('INHIBITOR_COOLDOWN', Math.ceil(existing.remainingTime / 1000), command.cooldownLevel !== 'author');
+		if (existing && existing.limited) throw message.language.get('INHIBITOR_COOLDOWN', this.secondsToFormattedString(Math.ceil(existing.remainingTime / 1000)), command.cooldownLevel !== 'author');
+	}
+
+	secondsToFormattedString(time) {
+		const days = Math.floor(time / 86400);
+		const hours = Math.floor(time / 3600);
+		const minutes = Math.floor(time / 3600 / 60);
+		const seconds = Math.floor(time / 3600 % 60);
+		return `${days ? `${days}d ` : ''}${hours ? `${hours}h ` : ''}${minutes ? `${minutes}m ` : ''}${seconds ? `${seconds}s ` : ''}`;
```

Comment:
Things should not just be able to pop out of existence if larger things are displayed... Also, this is wonky, try 2 days (172800) you get 2d 48h 48s (no minutes, and 48 seconds out of nowhere)  https://repl.it/repls/GrowingCourageousMisrac And you can't even get accurate seconds results.